app/uk/gov/hmrc/minorentityidentificationfrontend/views/check_your_answers_page.scala.html

```diff
@@ -30,7 +30,7 @@
 
 @(pageConfig: PageConfig, formAction: Call, summaryRows: Seq[SummaryListRow])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@layout(pageConfig, title = messages("check-your-answers.title")) {
+@layout(pageConfig, title = s"""${messages("check-your-answers.title")} - ${pageConfig.optServiceName.getOrElse(appConfig.defaultServiceName)} - ${messages("service.govuk")}""") {
```

Comment:
Hey Richard if we use this or something similar .... we can incapsulate all the logic of the new title in the titleHelper
<img width="1325" alt="Screenshot 2022-03-29 at 14 24 10" src="https://user-images.githubusercontent.com/4958044/160621270-5261520b-3d80-498c-af88-492a9dcd25c3.png">
r